nuxeo-runtime/nuxeo-runtime-test/src/main/java/org/nuxeo/runtime/test/runner/ConsoleLogThreshold.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * (C) Copyright 2019 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Salem Aouana
+ */
+
+package org.nuxeo.runtime.test.runner;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Annotation to define a console log level threshold on the root logger, which allow us to hide console log messages
+ * when launching tests.
+ * <p>
+ * The default level value is {@link java.util.logging.Level#OFF} which means all console log messages will be hidden.
+ *
```

Comment:
Can this be used on a class or just a method? Might be worth adding some details about that here, unless it's elsewhere.

Reply:
Yes the annotation can be used on Class or a method, i think it better to add mores details on the class java doc. Thx 👍